pom.xml

```diff
@@ -345,7 +310,89 @@
                     </execution>
                 </executions>
             </plugin>
-
         </plugins>
     </build>
+    <profiles>
+        <!-- The Configuration of the integration-test profile -->
+        <profile>
+            <id>integrationTest</id>
+            <properties>
+                <skipUTs>true</skipUTs>
+            </properties>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.0</version>
+                        <configuration>
+                            <skipTests>${skipTests}</skipTests>
+                            <skipITs>${skipITs}</skipITs>
+                            <systemPropertyVariables>
+                                <rabbit.host>localhost</rabbit.host>
```

Comment:
Should be named "rabbitmq.<key>" and not the pet/animal rabbit.

Reply:
Again it is not mine... result of merging... but can change it?

Reply:
I changed the name but it make integration tests fails... So I changed the name in integration test as well